_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title:  "How to build your own KafkaConnect image via custom resource"
+date: 2021-03-23
+author: jakub_stejskal
+---
+In one of [the older blog post](https://strimzi.io/blog/2020/05/07/camel-kafka-connectors/) we shown you how easy is to integrate Camel with Strimzi via KafkaConnect custom resource.
+That approach had one limitation - you had to build your own KafkaConnect image and use it in custom resource.
```

Comment:
I think you can use better `KafkaConnect` and Kafka Connect. I would use `KafkaConnect` when referring specifically to the custom resource (e.g. on the line above). But in places where you use it as generic component name I would just say Kafka Connect (with space, without formatting) - for example here or on the line below.